.ci_support/linux_64_.yaml

```
@@ -6,3 +6,5 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.10'
```

revert to bot's change (cf default)
```suggestion
- '3.9'
```